components/admin/requests/forms/PaymentDetailsForm.tsx

```diff
@@ -0,0 +1,384 @@
+import { PaymentDetails } from '@tools/components/admin/requests/forms/types';
+import {
+  FormControl,
+  FormLabel,
+  Input,
+  Text,
+  Stack,
+  FormHelperText,
+  Radio,
+  RadioGroup,
+  Box,
+  InputGroup,
+  InputLeftElement,
+  Checkbox,
+  Select,
+  Grid,
+  GridItem,
+  Divider,
+} from '@chakra-ui/react'; // Chakra UI
+import { PaymentType, Province } from '@lib/graphql/types';
+
+type PaymentDetailsFormProps = {
+  readonly paymentInformation: PaymentDetails;
+  readonly onChange: (updatedData: PaymentDetails) => void;
+};
+
+/**
+ * Function Passed into PaymentDetailsProps for monitoring the changes done to fields in PaymentDetailsForm Component.
+ *
+ * @callback onChangeCallback
+ * @param {PaymentDetails} updatedData
+ */
```

Comment:
This JSDoc doesn't describe anything (it needs to be on top of the thing it describes). Don't think we need it